contracts/RootChain.sol

```diff
@@ -205,7 +205,7 @@ contract RootChain {
      * @param _utxoPos The position of the exiting utxo in the format of blknum * 1000000000 + index * 10000 + oindex.
      * @param _txBytes The transaction being exited in RLP bytes format.
      * @param _proof Proof of the exiting transactions inclusion for the block specified by utxoPos.
-     * @param _sigs Both transaction signatures and confirmations signatures used to verify that the exiting transaction has been confirmed.
+     * @param _sigs Transaction signatures are needed to check tx inclusion.
```

Comment:
I think it's fine to remove `_sigs` entirely, assuming we also remove it from the Merkle hash: 

`bytes32 merkleHash = keccak256(keccak256(_txBytes), ByteUtils.slice(_sigs, 0, 130));`
=>
`bytes32 merkleHash = keccak256(_txBytes);`

Note that this also means making a change in the client to account for this. I believe only reason `_sigs` was part of the Merkle hash anyway was to account for confirmation signatures.

Reply:
This change is mostly a stop-gap solution before more-vp. I would prefer to avoid unnecessary work now.